src/engine/ruleset/decoders/syslog.yml

```diff
@@ -21,15 +21,18 @@ sources:
 parse:
   logpar:
     # BSD Syslog RFC 3164 standard
-    - event.original: "<event.start/Jun 14 15:16:01> <host.hostname> <process.name>[<process.pid>]:<~/ignore/ ><message>"
+    - event.original: "<event.start/Jun 14 15:16:01> <host.hostname> <process.name>[<process.pid>]: <~/ignore/ ><message>"
     # BSD Syslog RFC 3164 no pid
-    - event.original: "<event.start/Jun 14 15:16:01> <host.hostname> <process.name>:<~/ignore/ ><message>"
+    - event.original: "<event.start/Jun 14 15:16:01> <host.hostname> <process.name>: <~/ignore/ ><message>"
 
```

Comment:
The `ignore` here is to ignore the space, are there examples where 2 spaces can be present? In that case, maybe we should create new entries of `logpar` to catch them.